### ISSUE TITLE
Toggle Log panel with 'L' key

### ISSUE THREAD

**Issue by rappie** [crytic/echidna#1060]
### Describe the desired feature

The log panel is great but it can get in the way while debugging. It would be nice to have a toggle to hide it.

While you're at it, maybe also add a toggle for the tests panel in case someone wants to inspect the logs? :slightly_smiling_face:

**Comment by Yogesh01000100** [comment]
Is this issue solved, if not can you assign it to me?

**Comment by ggrieco-tob** [comment]
This is not solved yet, feel free to open a PR

**Comment by Yogesh01000100** [comment]
I'm facing issues while setting up the project locally, can you provide any specific recommended procedure for installation? I used wsl2.

**Comment by ggrieco-tob** [comment]
I don't think this was tested on wsl2 at all, even echidna itself. I'm afraid you will need an Linux/MacOS system for that.

**Comment by hriztam** [comment]
Hey, can I work on this issue ?

**Comment by arcz** [comment]
@hriztam yes
